lib/Conversion/HandshakeToHW/HandshakeToHW.cpp

```diff
@@ -1753,7 +1768,13 @@ class HandshakeToHWPass
                     ConvertToHWInstance<handshake::SIToFPOp>,
                     ConvertToHWInstance<handshake::FPToSIOp>,
                     ConvertToHWInstance<handshake::ExtFOp>,
-                    ConvertToHWInstance<handshake::AbsFOp>>(
+                    ConvertToHWInstance<handshake::AbsFOp>,
+                    // Operations for speculation
```

Comment:
comment indent looks weird (run clang format). it shouldn't touch the line above, add whitespace after the abs float op

Reply:
Clang format did not modify this line. Additionally, lines 1743 to 1745 also appear as follows:

```cpp
                    ConvertToHWInstance<handshake::SharingWrapperOp>,
                    // Arith operations
                    ConvertToHWInstance<handshake::AddFOp>,
```

I made manual changes to both lines.